spec/factories/image_works.rb

```diff
@@ -22,6 +22,13 @@
       end
     end
 
+    factory :image_work_with_one_metadata_file do
+      after(:create) do |image_work, evaluator|
+
+        1.times { image_work.ordered_members << FactoryGirl.create(:external_metadata_file, user: evaluator.user) }
```

Comment:
Looks like image_file,external_metadata_files, and raster_works all going into the same "ordered_members".  Should this just go into "members" or by some other means, as semantically there really isn't any order (ex: you could add the vector_work after the raster file or vice-versa, an one isn't really expected to come before another)